tests/unit/customizations/codeartifact/test_adapter_login.py

```diff
@@ -63,7 +63,6 @@ def test_run_commands_unhandled_error(self):
         with self.assertRaisesRegexp(OSError, 'unhandled error'):
             self.test_subject._run_commands(tool, ['echo', tool])
 
-
```

Comment:
Let's keep the newline here to follow pep8 practices of two newlines between classes (This happens again later on in the diff as well). In general, it would not hurt to run [`flake8`](https://flake8.pycqa.org/en/latest/) on the files being updated to make sure they follow pep8 style.

Reply:
Will do! Thanks